### DISCUSSION TITLE
Adding cells to a checkpointed fluid field

### DISCUSSION THREAD

**Discussion by mojtabaAmiraslanpour** [UvaCsl/HemoCell#71] [General discussion]
Hi,

Is there any way to load only the fluid field from the checkpoint and then add cells from the RBC.pos and PLT.pos files?

Thanks,
Moj

**Comment by gzavo** [comment]
Hi @mojtabaAmiraslanpour,

There is no ready-made solution for this. It shouldn't be too hard to implement this: grab the functions you need from the checkpointing mechanism, and call them in you own case file.
